apps/accounts/forms.py

```diff
@@ -377,8 +393,38 @@ class NetworkFootprintForm(MultiModelForm):
     form_classes = {
         "ips": IpRangeFormset,
         "asns": AsnFormset,
+        "extra": ExtraNetworkForm
     }
 
+    def clean(self):
+        """
+        Validate that a form at least has:
+        1. some IP and/or AS data
+        2. or some explanation
+
+        """
+        super().clean()
+
+        # fetch our forms
+        ip_form, asn_form, explanation_form = [form for form in self.forms.values()]
+
+        # is there any description we can read?
+        explanation_present = any(explanation_form.cleaned_data.values())
+
+        # do we have any in the two forms asking for network data?
+        network_data_present = ip_form.forms and asn_form.forms
```

Comment:
I think that an `or` condition is more appropriate here: we expect that either some IP ranges or ASNs are present. I also suggest we add a test case for that (IP form present, no ASN form present - or vice versa)